samples/doc_writer_sample_output.py

```diff
@@ -347,112 +359,112 @@
+                    "expects": "https://hydrus.com/api/extraClass",
                     "expectsHeader": [],
                     "method": "PUT",
                     "possibleStatus": [
                         {
                             "@context": "http://www.w3.org/ns/hydra/context.jsonld",
                             "@type": "Status",
-                            "description": "If the dummyClass entity was createdsuccessfully.",
+                            "description": "A new member in Extraclasses created",
                             "statusCode": 201,
                             "title": ""
                         }
                     ],
-                    "returns": "vocab:dummyClass",
+                    "returns": "https://hydrus.com/api/extraClass",
                     "returnsHeader": []
                 }
             ],
             "supportedProperty": [
                 {
                     "@type": "SupportedProperty",
-                    "description": "The dummyclass",
+                    "description": "The members of Extraclasses",
                     "property": "http://www.w3.org/ns/hydra/core#member",
                     "readable": "false",
                     "required": "false",
                     "title": "members",
                     "writeable": "false"
                 }
             ],
-            "title": "dummyClassCollection"
+            "title": "Extraclasses"
         },
         {
-            "@id": "vocab:extraClassCollection",
+            "@id": "https://hydrus.com/api/collection_2",
             "@type": "Collection",
-            "description": "A collection of extraclass",
+            "description": "This collection comprises of instances of dummyClass",
             "manages": {
-                "object": "vocab:extraClass",
+                "object": "https://hydrus.com/api/dummyClass",
                 "property": "rdf:type"
             },
             "subClassOf": "http://www.w3.org/ns/hydra/core#Collection",
             "supportedOperation": [
                 {
-                    "@id": "_:extraclass_collection_retrieve",
+                    "@id": "https://hydrus.com/api/collection_2/dummyclasses_retrieve",
                     "@type": "http://schema.org/FindAction",
-                    "description": "Retrieves all extraClass entities",
+                    "description": "Retrieves all the members of dummyclasses",
                     "expects": "null",
                     "expectsHeader": [],
                     "method": "GET",
                     "possibleStatus": [],
-                    "returns": "vocab:extraClassCollection",
+                    "returns": "https://hydrus.com/api/dummyClass",
                     "returnsHeader": []
                 },
                 {
-                    "@id": "_:extraclass_create",
+                    "@id": "https://hydrus.com/api/collection_2/dummyclasses_create",
```

Comment:
This is wrong. https://hydrus.com/api/collection_2/dummyclasses_create and https://hydrus.com/api/collection_2:dummyclasses_create are very different. When we use `/` instead of `:` we change the URI. Adding `:` is used to define a namespace, the URL would still be https://hydrus.com/api/collection_2